terms/experimentalData/otherMedicalDx.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "sage.annotations-experimentalData.otherMedicalDx-0.0.1",
+  "type": "string",
```

Comment:
If there might be multiple, perhaps we want this to be an array? I'm not sure whether JSON Schema arrays map to Synapse multi-value annotations -- if they work together harmoniously, this might be an option.

Reply:
When it's been used in individual human metadata in PsychENCODE it has been entered as a string value separated by commas.  We can make it an array, but then we'd have to go back and rework existing metadata.  It is not commonly used.

Reply:
Having it as an array seems valuable for queryability, but if it's not used often then maybe it's a bit moot.